### PR TITLE
[EDU-3181] feat: add records pt

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new page on Como adicionar registros.

[EDU-3181]: https://aziontech.atlassian.net/browse/EDU-3181?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/secure-jornada/intelligent-dns/edite-intelligent-dns/adicionar-registros.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+## Via RTM
+
+1. [Acesse o RTM](/pt-br/documentacao/produtos/guias/como-acessar-o-rtm/) > **Intelligent DNS**.
+2. Na lista, escolha o intelligent DNS no qual deseja adicionar registros.
+3. Selecione a aba **Records**.
+4. Clique no botão **Add Record**.
+5. Em **Name**, forneça o nome do novo registro. Dependendo do [tipo de registro](/pt-br/documentacao/produtos/intelligent-dns/#type) que você deseja usar, podem existir formatos e recomendações específicas. Por exemplo: para um registro CNAME, você pode usar o nome `www`.
+6. Em **Type**, selecione o tipo de registro que deseja adicionar. Aceita os tipos `A`, `AAAA`, `ANAME`, `CAA`, `CNAME`, `DS`, `MX`, `NS`, `PTR`, `SRV` e `TXT`.
+7. Em **Value**, adicione os itens para a resposta DNS ao registro registrado. Os valores aceitos variam de acordo com o [tipo escolhido de registro](/pt-br/documentacao/produtos/intelligent-dns/#type).
+8. Em **TTl (segundos)**, escolha o tempo, em segundos, que uma resposta pode ser armazenada em um servidor de resolução. Valor máximo: `2147483647`.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/secure-jornada/intelligent-dns/edite-intelligent-dns/adicionar-registros.mdx:23]:
```suggestion
8. Em **TTL (seconds)**, escolha o tempo, em segundos, que uma resposta pode ser armazenada no servidor de resolução. Valor máximo: `2147483647`.
```

Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/secure-jornada/intelligent-dns/edite-intelligent-dns/adicionar-registros.mdx:23]:
```suggestion
8. Em **TTL (seconds)**, escolha o tempo, em segundos, que uma resposta pode ser armazenada em um servidor de resolução. Valor máximo: `2147483647`.
```

**Review by gabriel-azion — APPROVED** [review]

File: src/content/docs/pt-br/pages/secure-jornada/intelligent-dns/edite-intelligent-dns/adicionar-registros.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+title: Como adicionar registros
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/secure-jornada/intelligent-dns/edite-intelligent-dns/adicionar-registros.mdx:2]:
```suggestion
title: Como adicionar registros em uma zona de DNS
```

**Review by bru-andrade — APPROVED** [review]
✨

**Pull request merged**